logging/logrus/client_interceptors_test.go

```diff
@@ -187,3 +187,40 @@ func (s *logrusClientOverrideSuite) TestPingList_HasOverrides() {
 	assert.NotContains(s.T(), msgs[0], "grpc.time_ms", "message must not contain default duration")
 	assert.Contains(s.T(), msgs[0], "grpc.duration", "message must contain overridden duration")
 }
+
+func TestZapLoggingClientMessageProducerSuite(t *testing.T) {
+	if strings.HasPrefix(runtime.Version(), "go1.7") {
+		t.Skip("Skipping due to json.RawMessage incompatibility with go1.7")
```

Comment:
I wonder how long it will take for us to get to `go1.70.0` :thinking:

Reply:
Lets remove this, we don't support `Go 1.7`

Reply:
okay 😄 

i will remove it in the files i've touched. The PR would get messy otherwise ^^